packages/core/src/sass/app/_common/layout/modal-root.scss

```diff
@@ -8,7 +8,7 @@
     left: 0;
     bottom: 0;
     right: 0;
-    z-index: 9999;
+    z-index: 999;
```

Comment:
Why was this reduced?

Reply:
I need a popover to be shown on top of the modal. Currently, popover has a z-index of 999 which will appear under the modal.

Reply:
[Resolved. We increased `z-index` of popover instead.]